batch-jberet/src/main/resources/org/wildfly/extension/batch/jberet/LocalDescriptions.properties

```diff
@@ -50,7 +50,7 @@ batch.jberet.jdbc-job-repository.execution-records-limit=The maximum number of j
   be returned by the job repository.
 
 # Thread pool
-batch.jberet.thread-pool=The thread pool used for batch jobs. Note that the max-thread attribute should always be greater \
+batch.jberet.thread-pool=The thread pool is used for batch jobs. Note that the max-thread attribute should always be greater \
```

Comment:
I think the original sentence is fine. It means: this is the thread pool used for batch jobs.

Reply:
@moulalis Can you correct this please? I'd like to get this merged. Thanks.

Reply:
> @moulalis Can you correct this please? I'd like to get this merged. Thanks.

@bstansberry Done. Thanks.